app/views/devise/sessions/new.html.slim

```diff
@@ -19,7 +16,7 @@ p.my3 == t('notices.log_in_consent.text', app: APP_NAME, link: link)
 .clearfix.pt1.border-top
   - if @sp_name && @sp_return_url
```

Comment:
Could this conditional be removed as well by setting `back_to_service_provider_link` to `nil` for SessionDecorator? Or is that confusing because there is no service provider in that context?